scripts/entrypoint.sh

```diff
@@ -8,4 +8,8 @@ if [ -n "${USER_UID}" ]; then
   exec su user -c "${*@Q}"
 fi
 
+if [ -n "${TF_VERSION}" ]; then
+  tfswitch "${TF_VERSION}"
+fi
+
```

Comment:
this does not ensure to be run with the latest available..?

Reply:
nope - this ensures that if `TF_VERSION` is set, tfswitch will switch to the version that is defined in `TF_VERSION`. `TF_VERSION` is currently set to `1.0` in https://github.com/mineiros-io/build-tools/blob/dc55ac5c5af8e4946e6d8a1bcb7b6c23aac40cfc/Makefile#L8. When building the Docker image, we are calling `tfswitch $TF_VERSION --latest-stable` (see https://github.com/mineiros-io/build-tools/blob/dc55ac5c5af8e4946e6d8a1bcb7b6c23aac40cfc/Dockerfile#L78 for details), it will install the latest stable version of the `1.x` version

Reply:
so imho, it doesn exactly what we want. If you e.g. set `TF_VERSION` on runtime, tfswitch will switch to whatever version is provided. `TF_VERSION` in the Dockerfile is a build arg and not an environment variable.